### ISSUE TITLE
How to show one label only even though it detects all the objects

### ISSUE THREAD

**Issue by rezaabdullah** [thtrieu/darkflow#618]
Hi Thtrieu,

How to show only one label? Let's say for example,
I am using tiny-yolo model which detects 20 object classes. However instead of showing bounding box i.e. labels for all the object classes, is it possible to show only one object class? Let's say a picture contains Persons, dog, bicycle and cars. Once done it only shows bounding box for the person although it detects all the other objects.

**Comment by idhamhalim** [comment]
You should change the label.txt and the cfg file (should be the same name as your model) based on how many labels that you have set

https://github.com/thtrieu/darkflow#training-on-your-own-dataset

**Comment by rezaabdullah** [comment]
Hi Idham,

Thanks for the quick response. One concern is I am not planning to train my model. My objective is as follows:

I am planning to create a traffic monitoring system for a shop. So I need to know how many person are going in and coming out. Although, I can get the number of person from JSON output, it is more convincing if I can show that the system detects only people and differentiate them from other objects.

Thank you.

Regards,
Abdullah Reza

________________________________

**Comment by troydeng** [comment]
[](url)https://github.com/thtrieu/darkflow/blob/master/darkflow/net/yolov2/predict.py
check this file, and just add a little piece of code in the postprocess function
```
def postprocess(self, net_out, im, save=True):
    """
	Takes net output, draw net_out, save to disk
	"""
    boxes = self.findboxes(net_out)

    # meta
    meta = self.meta
    threshold = meta['thresh']
    colors = meta['colors']
    labels = meta['labels']
    if type(im) is not np.ndarray:
        imgcv = cv2.imread(im)
    else:
        imgcv = im
    h, w, _ = imgcv.shape

    resultsForJSON = []
    for b in boxes:
        boxResults = self.process_box(b, h, w, threshold)
        if boxResults is None:
            continue
        left, right, top, bot, mess, max_indx, confidence = boxResults

        #################################################
        ##added by me
        if max_indx != 14:
            continue
        #################################################

        thick = int((h + w) // 300)
        if self.FLAGS.json:
```
I hope this would help you, hahaha

**Comment by rezaabdullah** [comment]
Hi,

Can't thank you enough. Really appreciate your help.

-Reza
________________________________

**Comment by mursyidinadnan** [comment]
@troydeng where is the file i should place this code? 
`labels20 = ["aeroplane", "bicycle", "bird", "boat", "bottle",
    "bus", "car", "cat", "chair", "cow", "diningtable", "dog",
    "horse", "motorbike", "person", "pottedplant", "sheep", "sofa",
    "train", "tvmonitor"]`

**Comment by mursyidinadnan** [comment]
hi @rezaabdullah is that code 
###########
if max_indx != 14: 
  continue
########
is work for you? because when i runinng with that code all box is not display
please if you can solve that case? thankyou

**Comment by rezaabdullah** [comment]
Hi,

Yes the code works perfectly. The file path is as follows:
https://github.com/thtrieu/darkflow/blob/master/darkflow/net/yolo/predict.py
[https://avatars2.githubusercontent.com/u/9194247?s=400&v=4]<https://github.com/thtrieu/darkflow/blob/master/darkflow/net/yolo/predict.py>

darkflow/predict.py at master · thtrieu/darkflow · GitHub<https://github.com/thtrieu/darkflow/blob/master/darkflow/net/yolo/predict.py>
darkflow - Translate darknet to tensorflow. Load trained weights, retrain/fine-tune using tensorflow, export constant graph def to mobile devices
github.com


________________________________

**Comment by rezaabdullah** [comment]
@mursyidinadnan : pls edit the following file:
https://github.com/thtrieu/darkflow/blob/master/darkflow/net/yolov2/predict.py

**Comment by StraigenDaigen** [comment]
I did the steps, but when I run my classification script in real time, I followed the other tags classified.
![image](https://user-images.githubusercontent.com/43056610/45199499-9ce20580-b231-11e8-9d80-c1cc843e0800.png)
![image](https://user-images.githubusercontent.com/43056610/45199544-dc105680-b231-11e8-8670-63e4ebe49990.png)
I need to classify only "person".
